docs/dxp/latest/en/users-and-permissions/developer-guide/api/document-api-basics/resources/liferay-y6q4.zip/curl/User_PATCH_ById.sh

```diff
@@ -0,0 +1,5 @@
+curl \
+-H "Content-Type: application/json" \
+-X PATCH "http://localhost:8080/o/headless-admin-user/v1.0/user-accounts/${1}" \
+-d "{\"alternateName\":\"Bar\",\"emailAddress\":\"bar@liferay.com\",\"familyName\":\"Liferay\",\"givenName\":\"Bar\"}" \
```

Comment:
Use a space before each key or value that follows a delimeter (e.g., after each `:` and `,` in this file).